src/main/java/net/minecraftforge/server/command/CommandSetDimension.java

```diff
@@ -0,0 +1,148 @@
+package net.minecraftforge.server.command;
+
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.command.WrongUsageException;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.world.Teleporter;
+import net.minecraft.world.WorldServer;
+import net.minecraftforge.common.DimensionManager;
+import net.minecraftforge.common.ForgeHooks;
+
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.WeakHashMap;
+
+public class CommandSetDimension extends CommandBase
+{
+    @Override
+    public String getName()
+    {
+        return "setdimension";
+    }
+
+    @Override
+    public List<String> getAliases()
+    {
+        return Collections.singletonList("setdim");
```

Comment:
`tp` might be another nice alias for this as well

Reply:
I picked a different name to distinguish this from the vanilla `tp` and `teleport` commands, as it's mainly intended for something else.

If it's wanted, I can add more aliases, though.